docs/geedocs/answer/1090589.html

```diff
@@ -28,7 +28,7 @@
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Fusion resources and projects</h1>
 <h2>Import a subset of imagery</h2>
 <p>You can import a single area of a source image to Fusion as a new, smaller image file. For example, shown below is a compressed, high-resolution MrSID source image of Santa Clara County, California. On a standard server machine, it would take several hours to import this entire image to Fusion.</p>
-<p>If you don't need the entire image, you can select an area of the image to import, then use <code>getranslate -projwin</code> to extract the area and create a new image from it. The <code>-projwin</code> parameter reads the coordinates of the bounding box you specify and writes out your selection as a new image. The coordinates are based on the native projection in the source image. You can use Google Earth to easily find corner coordinates for the area you want to copy. (To learn how, see <a href="https://support.google.com/earth/answer/148070" target="_blank">Viewing preferences</a> in the <em>Google Earth Help</em>.) The <code>getranslate</code> tool, extended from the GDAL <code>gdal_translate</code> tool, includes support for MrSID and JPEG2000 format source imagery.</p>
+<p>If you don't need the entire image, you can select an area of the image to import, then use <code>getranslate -projwin</code> to extract the area and create a new image from it. The <code>-projwin</code> parameter reads the coordinates of the bounding box you specify and writes out your selection as a new image. The coordinates are based on the native projection in the source image. You can use Google Earth to easily find corner coordinates for the area you want to copy. (To learn how, see <a href="https://support.google.com/earth/answer/148068" target="_blank">Find & use location coordinates</a> in the <em>Google Earth Help</em>.) The <code>getranslate</code> tool, extended from the GDAL <code>gdal_translate</code> tool, includes support for MrSID and JPEG2000 format source imagery.</p>
```

Comment:
This is a separate issue, but we might as well fix it here - we need to remove the statement that gdal_translate supports MrSID.